libs/designsystem/chart/src/stock-chart/stock-chart-js.service.ts

```diff
@@ -30,7 +30,7 @@ export class StockChartJSService extends ChartJSService {
     dataLabelOptions?: ChartDataLabelOptions;
     highlightedElements?: ChartHighlightedElements;
   }): void {
-    const { dataLabelOptions, ...superArgs } = args;
+    const { ...superArgs } = args;
     this.dataLabelOptions = args.dataLabelOptions || null;
```

Comment:
Not sure if it matters, but this means `dataLabelOptions` is now destructured into `superArgs` - maybe leave as is and use the variable below?
```suggestion
    const { dataLabelOptions, ...superArgs } = args;
    this.dataLabelOptions = dataLabelOptions || null;
```

Reply:
Oh wow, nice catch. My change reads poorly and who knows what funky behavior might result from it 😅